src/modules/cmd_duration.rs

```diff
@@ -0,0 +1,93 @@
+use crate::config::Config;
+use ansi_term::Color;
+
+use super::{Context, Module};
+
+/// Outputs the time it took the last command to execute
+///
+/// Will only print if last command took more than a certain amount of time to
+/// execute. Default is two seconds, but can be set by config option `min_time`.
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("cmd_duration")?;
+    let arguments = &context.arguments;
+    let elapsed = arguments
+        .value_of("cmd_duration")
+        .unwrap_or("invalid_time")
+        .parse::<u64>()
+        .ok()?;
+
+    let signed_config_min = context
```

Comment:
The `config_value_X` convenience functions should work here:

```rust
let signed_config_min = module
	.config_value_i64("min_time")
	.unwrap_or(2);
```

Reply:
Oh yeah, that's much better.

Was this added recently or did I miss it when I was writing this?

Reply:
Yeah, it was a recent addition in #120.